StormCPackConfig.cmake

```diff
@@ -12,10 +12,10 @@ set(CPACK_PACKAGE_CONTACT "support@stormchecker.org")
 set(CPACK_RESOURCE_FILE_LICENSE "${CMAKE_SOURCE_DIR}/LICENSE")
 
 ### versions
-set(CPACK_PACKAGE_VERSION_MAJOR "${STORM_CPP_VERSION_MAJOR}")
-set(CPACK_PACKAGE_VERSION_MINOR "${STORM_CPP_VERSION_MINOR}")
-set(CPACK_PACKAGE_VERSION_PATCH "${STORM_CPP_VERSION_PATCH}")
-set(CPACK_PACKAGE_VERSION "${CPACK_PACKAGE_VERSION_MAJOR}.${CPACK_PACKAGE_VERSION_MINOR}.${CPACK_PACKAGE_VERSION_PATCH}-${STORM_CPP_VERSION_HASH}")
+set(CPACK_PACKAGE_VERSION_MAJOR "${STORM_VERSION_MAJOR}")
+set(CPACK_PACKAGE_VERSION_MINOR "${STORM_VERSION_MINOR}")
+set(CPACK_PACKAGE_VERSION_PATCH "${STORM_VERSION_PATCH}")
+set(CPACK_PACKAGE_VERSION "${CPACK_PACKAGE_VERSION_MAJOR}.${CPACK_PACKAGE_VERSION_MINOR}.${CPACK_PACKAGE_VERSION_PATCH}-${STORM_VERSION_HASH}")
```

Comment:
Sorry to complain, but `STORM_VERSION_HASH` does not exist. Maybe just remove this line, because `CPACK_PACKAGE_VERSION` is already [created by default](https://cmake.org/cmake/help/latest/module/CPack.html#variable:CPACK_PACKAGE_VERSION).